brokerapi/brokers/bigquery/broker.go

```diff
@@ -39,44 +39,39 @@ type InstanceInformation struct {
 
 // Provision creates a new BigQuery dataset from the settings in the user-provided details and service plan.
 func (b *BigQueryBroker) Provision(ctx context.Context, instanceId string, details brokerapi.ProvisionDetails, plan models.ServicePlan) (models.ServiceInstanceDetails, error) {
-	var err error
-	var params map[string]string
+	variableContext, err := varcontext.Builder().
+		MergeDefaults(serviceDefinition().ProvisionInputVariables).
```

Comment:
Is this the very specif order you call out in the PR comments? `(operator defined service plan vars -> user vars -> default vars)`

Is this ordering documented in a `.md` file somewhere?